contexts/auth-context.provider.tsx

```diff
@@ -95,22 +107,22 @@ export const AuthContextProvider: React.FC<PropsWithChildren> = ({
 
       setSession(userCredential.user.uid);
 
-      await fetch('https://be.trendly.pro/api/v1/chat/auth', {
-        method: 'POST',
+      await fetch("https://be.trendly.pro/api/v1/chat/auth", {
+        method: "POST",
         headers: {
-          'Content-Type': 'application/json',
-          'Authorization': `Bearer ${userCredential.user.uid}`,
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${userCredential.user.uid}`,
         },
       });
 
-      await analyticsLogEvent('signed_in', {
+      await analyticsLogEvent("signed_in", {
         id: userCredential.user.uid,
         name: userCredential.user.displayName,
         email: userCredential.user.email,
       });
 
       // For existing users, redirect to the main screen.
-      router.replace("/collaborations");
+      // router.replace("/collaborations");
```

Comment:
Why did we comment on this?

Reply:
Because now we want to first check if user has socials then only we will move then to collaborations

Reply:
So can we remove it instead of commenting?

Reply:
Ok